QModManager/Patching/Patcher.cs

```diff
@@ -54,18 +54,34 @@ internal static void Patch()
                     return;
                 }
 
+
                 try
                 {
-                    Logger.Info("Folder structure:");
+                    Logger.Info("Game Folder structure:");
                     IOUtilities.LogFolderStructureAsTree();
-                    Logger.Info("Folder structure ended.");
+                    Logger.Info("Game Folder structure ended.");
                 }
                 catch (Exception e)
                 {
                     Logger.Error("There was an error while trying to display the folder structure.");
                     Logger.Exception(e);
                 }
 
+                if (!QModBaseDir.StartsWith(Environment.CurrentDirectory))
```

Comment:
Be careful with this, it will sometimes give you false negatives.
https://stackoverflow.com/a/23354773

Reply:
a2b4610